app/scripts/components/common/catalog/utils.test.ts

```diff
@@ -0,0 +1,104 @@
+import { omit, set } from 'lodash';
+import { CatalogActions, onCatalogAction } from './utils';
+
+describe('onCatalogAction', () => {
+  let setSearchMock;
+  let setTaxonomiesMock;
+  let taxonomies;
+
+  beforeEach(() => {
+    setSearchMock = jest.fn();
+    setTaxonomiesMock = jest.fn();
+    taxonomies = {
+      Topics: ['air-quality', 'climate'],
+      Sectors: ['electricity', 'energy']
+    };
+  });
+
+  it('should clear search and taxonomies on CLEAR action', () => {
```

Comment:
I was wondering if this behavior is intended or not. Great to see it on test to clarify the expected behavior

Reply:
It's matching prod, but the CLEAR should probably only remove the tags (the input already has a clear button by itself). Will leave it in for now to match prev behavior.